src/utils/reporter.ts

```diff
@@ -34,3 +35,17 @@ export function reportSuccess(classesInfo: GenezioCloudResultClass[]) {
         log.info(printHttpString);
     }
 }
+
+export function reportSuccessFunctions(functions: DeployCodeFunctionResponse[]) {
+    let printHttpString = "";
```

Comment:
Please rename this variable to reflect that it contains functions, not HTTP.
```suggestion
    let functionDeployments = "";
```